src/Categories/Category/Cartesian/Structure.agda

```diff
@@ -0,0 +1,20 @@
+{-# OPTIONS --without-K --safe #-}
+
+-- here we define the structure version of a cartesian category
+module Categories.Category.Cartesian.Structure where
```

Comment:
Is this a pattern we'd like to see elsewhere in the library? For example, for monoidal categories?

Reply:
there was some debate going on, but I think it's good to have this pattern around: https://github.com/agda/agda-categories/issues/106